packages/tools/src/hash/multibase-encoded-256.ts

```diff
@@ -13,12 +13,16 @@ export class MultibaseEncodedSHA256 implements IAgentPlugin {
     };
   }
 
-  async computeHash(value: string): Promise<string> {
-    if (!value || typeof value !== 'string') {
+  async computeHash(args: IToolsComputeHashArgs): Promise<string> {
+    if (
+      typeof args !== 'object' ||
+      !args?.value ||
+      typeof args.value !== 'string'
+    ) {
       throw new Error('Value is invalid');
     }
```

Comment:
Could you explain why we should add additional checks for the input arguments? We already have an interface for the input arguments, so I think we only need to check if the `args.value` property is an empty string.

Reply:
Hi @huynguyen-hl, the interface I think only checks during the compiler code and it doesn't exist when the code is running. 

The image below: when a user does not input anything and the code also do not check anything:
<img width="1049" alt="image" src="https://github.com/uncefact/project-vckit/assets/59736798/9748c7bb-cf4f-43c5-8874-ce7aa73f05e8">